frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.tsx

```diff
@@ -73,7 +71,6 @@ export function LemonTabs<T extends string | number>({
             className={clsx(
                 'LemonTabs',
                 transitioning && 'LemonTabs--transitioning',
-                inline && 'LemonTabs--inline',
                 borderless && 'LemonTabs--borderless'
```

Comment:
We also don't need borderless right?

Reply:
We don't but @Gilbert09 started using it somewhere else 🙈 I was going to follow up with him and find an alternative once he's back